src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -157,11 +160,23 @@ public void doMove(final Map<Territory, ProTerritory> attackMap, final IMoveDele
     ProMoveUtils.calculateAmphibRoutes(player, moveUnits, moveRoutes, transportsToLoad, attackMap, true);
     ProMoveUtils.doMove(moveUnits, moveRoutes, transportsToLoad, moveDel);
 
-    // Calculate attack routes and perform moves
+    // Calculate bombard routes and perform moves
     moveUnits.clear();
     moveRoutes.clear();
     ProMoveUtils.calculateBombardMoveRoutes(player, moveUnits, moveRoutes, attackMap);
     ProMoveUtils.doMove(moveUnits, moveRoutes, null, moveDel);
+
+    // Calculate bombing routes and perform moves
+    moveUnits.clear();
```

Comment:
Yikes, this is a lot of state we are managing for other objects. Can we avoid any of this data state?